docs/templates/function.js

```diff
@@ -0,0 +1,143 @@
+import hljs from "highlight.js";
+import { promises } from "fs";
+
+import { outPath } from "../utils/io.js";
+import { getPath } from "../utils/url.js";
+import { scrollTo } from "../utils/scripts.js";
+import pageTemplate from "./page.js";
+import last from "../../array/last.js";
+
+const { writeFile: writeFileAsync, mkdir: mkdirAsync } = promises;
+
+const breadcrumb = (name, pathParts) => {
+  const parts = ["", ...pathParts, name];
+
+  let resultParts = [];
+
+  for (let i = 1; i < parts.length; i++) {
+    const part = parts.slice(0, i);
+    resultParts.push(
+      `<a href="${getPath(part)}">${last(part) === "" ? "std" : last(part)}</a>`
```

Comment:
Please extract sub-elements as constants - this way the template would be much simpler and could be extracted as `const breadcrumbPart(anchor, text) => ...`.